dbt/adapters/athena/python_submissions.py

```diff
@@ -0,0 +1,123 @@
+import time
+from functools import cached_property
+from typing import Any, Dict
+
+import boto3
+
+from dbt.adapters.athena.connections import AthenaCredentials
+from dbt.adapters.base import PythonJobHelper
+from dbt.events import AdapterLogger
+from dbt.exceptions import DbtRuntimeError
+
+DEFAULT_POLLING_INTERVAL = 2
+SUBMISSION_LANGUAGE = "python"
+DEFAULT_TIMEOUT = 60 * 60 * 24
+
+logger = AdapterLogger("Athena")
+
+
+class AthenaPythonJobHelper(PythonJobHelper):
+    def __init__(self, parsed_model: Dict, credentials: AthenaCredentials) -> None:
+        self.identifier = parsed_model["alias"]
+        self.schema = parsed_model["schema"]
+        self.parsed_model = parsed_model
+        self.timeout = self.get_timeout()
+        self.polling_interval = DEFAULT_POLLING_INTERVAL
+        self.region_name = credentials.get_region_name()
+        self.profile_name = credentials.get_profile_name()
+        self.spark_work_group = credentials.get_spark_work_group()
+
+    @cached_property
+    def session_id(self) -> str:
+        if self._list_sessions() is None:
+            return self._start_session().get("SessionId")
+        return self._list_sessions().get("SessionId")
+
+    @cached_property
+    def athena_client(self) -> Any:
+        return boto3.client("athena")
```

Comment:
instead of using boto3.client consider to use https://github.com/dbt-athena/dbt-athena/blob/24e2a300072b311d29a352e4a8a249a0975ba0c3/dbt/adapters/athena/session.py#L6-L10

Reply:
How do I pass that connection object? Is that the same as AthenaConnectionManager?

Reply:
Yeah I guess so, alternatively create a new function that return a boto3 session with the region and aws profile name passed from the target profiles.

Reply:
I couldn't create the connection object but I was able to create a session using the `AthenaCredentials` as suggested